generator.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const faker = require('faker')
+const argv = require('yargs').argv
+const helpers = require('./helpers/helpers.js')
+
+const residences = argv.residences || 10000000
+const filename = argv.output || 'residences.csv'
+const stream = fs.createWriteStream(filename)
+
+const createResidence = () => {
+  const residenceName = faker.company.companyName();
+  const price = faker.commerce.price();
+  const url = helpers.generateRandomImage();
```

Comment:
Two things come to mind for this:
1) Am I correct in interpreting that this means each residence will only ever one photo attached to it?
2) I think you could be slightly more precise in naming this function. You're not generating a random image exactly, you're generating a random number that corresponds to the URLs of images in your S3 database.